packages/mobile/src/pages/ScreensAuthorization/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import {
+  KeyboardAvoidingView,
+  ScrollView,
+  View,
+  Image,
+  TouchableWithoutFeedback,
+  Platform,
+  Keyboard,
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import { InterfaceStore } from './../../../store';
+import { AUTH_LOGIN, PayloadAuthLogin } from './../../../store/auth/actions';
+import { SignInState, AuthorizationProps } from './../interfaces';
+import { validateInput, hideMessage } from './../helpers';
+import styles from './styles';
+import { CustomButton } from './../../../components/Button';
+import { CustomInput } from './../../../components/Input';
+import { Divider } from './../../../components/Divider';
+import { SocialLogin } from './../../../components/SocialLogin';
+import { NotifyMessage } from './../../../components/NotifyMessage';
```

Comment:
Can we avoid such './../../../' imports somehow?

Reply:
Yes, I will fix imports like this in future requests.